test/sample-test/sample_test_launcher.py

```diff
@@ -200,12 +200,12 @@ def __init__(self, test_name, results_gcs_dir,
   def _injection(self):
     """Sample-specific image injection into yaml file."""
     subs = {
-        'gcr\.io/ml-pipeline/ml-pipeline/ml-pipeline-local-confusion-matrix:\w+':self._local_confusionmatrix_image,
-        'gcr\.io/ml-pipeline/ml-pipeline/ml-pipeline-local-roc:\w+':self._local_roc_image
+        'gcr\.io/ml-pipeline/ml-pipeline/ml-pipeline-local-confusion-matrix:(\w+|[.])+':self._local_confusionmatrix_image,
+        'gcr\.io/ml-pipeline/ml-pipeline/ml-pipeline-local-roc:(\w+|[.])+':self._local_roc_image
```

Comment:
Does this capture '-'?

Reply:
Thanks! That's right, it doesn't capture it. Let me do a quick fix.